fighthealthinsurance/common_view_logic.py

```
@@ -1657,9 +1681,8 @@ async def extract_set_denialtype(cls, denial_id):
                 await DenialTypesRelation.objects.acreate(
                     denial=denial, denial_type=dt, src=await cls.regex_src()
                 )
-            except Exception as e:
-                # Can fail if relation already exists (duplicate)
-                logger.opt(exception=True).debug(f"Failed setting denial type: {e}")
+            except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)